Packs/CortexAttackSurfaceManagement/Playbooks/Cortex_ASM_-_ASM_Alert.yml

```diff
@@ -2888,42 +3025,40 @@ tasks:
     quietmode: 0
     isoversize: false
     isautoswitchedtoquietmode: false
-  "120":
-    id: "120"
-    taskid: 03f0b59b-af2b-4ede-8bda-aa7857c79c82
-    type: playbook
+  "131":
+    id: "131"
+    taskid: 82f16b4f-24ef-42e1-8e88-81ee4fbc9d56
+    type: regular
     task:
-      id: 03f0b59b-af2b-4ede-8bda-aa7857c79c82
+      id: 82f16b4f-24ef-42e1-8e88-81ee4fbc9d56
       version: -1
-      name: Cortex ASM - Remediation Path Rules
-      playbookName: Cortex ASM - Remediation Path Rules
-      type: playbook
+      name: Set playbook stage grid field (decision)
+      description: |-
+        Automation used to more easily populate a grid field.  This is necessary when you want to assign certain values as static or if you have context paths that you will assign to different values as well.  Instead of a value you can enter `TIMESTAMP` to get the current timestamp in ISO format.  Example of command:
```

Comment:
```suggestion
        Automation used to more easily populate a grid field. This is necessary when you want to assign certain values as static or if you have context paths that you will assign to different values as well. Instead of a value, you can enter `TIMESTAMP` to get the current timestamp in ISO format.  Example of the command:
```